src/app/landing-page/landing-page.component.scss

```diff
@@ -0,0 +1,42 @@
+.container {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header {
+  font-size: 2.9rem;
+  margin: 1.6rem 0 0.8rem 0;
+}
+
+.dot-container {
+  display: flex;
+  justify-self: flex-end;
+  margin: 1rem;
+}
+
+.btn-container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  margin-bottom: 2.4rem;
+}
+
+.img-container {
+
+}
+
+.dot {
+  background-color: #adadad;
+  width: 0.4rem;
+  height: 0.4rem;
+  border-radius: 10rem;
```

Comment:
Good practice is to set border-radius just as much as required, not more, because that might lead to some weird ellipsoid shapes on some devices or browsers. Not really common, but still, it's good practice. If there is a need for circle, border-radius should be 50% of size, so 0.2rem in this case.